pkg/bootstrap/option/instances_test.go

```diff
@@ -204,6 +204,37 @@ func TestOptions(t *testing.T) {
 			option:   option.LightstepToken("fake"),
 			expected: "fake",
 		},
+		{
+			testName: "openCensusAgent address",
+			key:      "openCensusAgent",
+			option:   option.OpenCensusAgentAddress("fake-ocagent"),
+			expected: "fake-ocagent",
+		},
+		{
+			testName: "openCensusAgent empty context",
+			key:      "openCensusAgentContexts",
+			option:   option.OpenCensusAgentContexts([]meshAPI.Tracing_OpenCensusAgent_TraceContext{}),
+			expected: "[\"TRACE_CONTEXT\",\"GRPC_BIN\",\"CLOUD_TRACE_CONTEXT\",\"B3\"]",
+		},
+		{
+			testName: "openCensusAgent order context",
+			key:      "openCensusAgentContexts",
+			option: option.OpenCensusAgentContexts([]meshAPI.Tracing_OpenCensusAgent_TraceContext{
+				meshAPI.Tracing_OpenCensusAgent_CLOUD_TRACE_CONTEXT,
+				meshAPI.Tracing_OpenCensusAgent_B3,
+				meshAPI.Tracing_OpenCensusAgent_GRPC_BIN,
+				meshAPI.Tracing_OpenCensusAgent_W3C_TRACE_CONTEXT,
+			}),
+			expected: "[\"CLOUD_TRACE_CONTEXT\",\"B3\",\"GRPC_BIN\",\"TRACE_CONTEXT\"]",
```

Comment:
Sorry missed this earlier, but you could use back quotes everywhere.

Reply:
Done. You are right, I should have looked for all usage when you made that comment. Thanks!